spec/integration/test.js

```diff
@@ -17,82 +17,94 @@ describer('notification api with a live service', () => {
-    var sms_notification_json = require('./schemas/v0/sms_notification.json');
-
+    var definitions_json = require('./schemas/v2/definitions.json');
     chai.tv4.addSchema('definitions.json', definitions_json);
-    chai.tv4.addSchema('email_notification.json', email_notification_json);
-    chai.tv4.addSchema('sms_notification.json', sms_notification_json);
 
   });
 
   describe('should send', () => {
-    var post_notification_return_email_json = require('./schemas/v0/POST_notification_return_email.json');
+
     it('email', () => {
+
+      var post_notification_return_email_json = require('./schemas/v2/POST_notification_email_response.json');
       const emailTemplateId = process.env.EMAIL_TEMPLATE_ID;
-      return notifyClient.sendEmail(emailTemplateId, email, personalisation).then((response) => {
-        expect(response.body).to.be.jsonSchema(post_notification_return_email_json);
+      return notifyClient.sendEmail(emailTemplateId, email, personalisation, clientRef).then((response) => {
         response.statusCode.should.equal(201);
-        response.body.data.notification.should.have.property('id');
-        response.body.data.body.should.equal('Hello Foo\n\nFunctional test help make our world a better place');
-        response.body.data.subject.should.equal('Functional Tests are good');
-        emailNotificationId = response.body.data.notification.id
-
+        expect(response.body).to.be.jsonSchema(post_notification_return_email_json);
+        response.body.content.body.should.equal('Hello Foo\n\nFunctional test help make our world a better place');
+        response.body.content.subject.should.equal('Functional Tests are good');
+        response.body.reference.should.equal(clientRef);
+        emailNotificationId = response.body.id;
       });
+
     });
 
     it('sms', () => {
-      var post_notification_return_sms_json = require('./schemas/v0/POST_notification_return_sms.json');
+
+      var post_notification_return_sms_json = require('./schemas/v2/POST_notification_sms_response.json');
       const smsTemplateId = process.env.SMS_TEMPLATE_ID;
       return notifyClient.sendSms(smsTemplateId, phoneNumber, personalisation).then((response) => {
+        response.statusCode.should.equal(201);
         expect(response.body).to.be.jsonSchema(post_notification_return_sms_json);
-        response.body.data.notification.should.have.property('id');
-        response.body.data.body.should.equal('Hello Foo\n\nFunctional Tests make our world a better place');
-        response.statusCode.should.equal(201)
-        smsNotificationId = response.body.data.notification.id;
+        response.body.content.body.should.equal('Hello Foo\n\nFunctional Tests make our world a better place');
+        smsNotificationId = response.body.id;
       });
+
     });
+
   });
 
   describe('should retrieve', () => {
 
+    var notification_json = require('./schemas/v2/GET_notification_response.json');
```

Comment:
even more minor quibble (esp. as in a test) but would be good to use either camel case or snake case